zqd/api/jsonpipescanner.go

```diff
@@ -0,0 +1,33 @@
+package api
```

Comment:
Why is this in package api, when it's only used during package zqd testing?

Reply:
When I add back the `zqdcli` pr that I was working on, the client functionality will live in `api` and use this. This is the humble beginnings of the client api.